recipes/optuna-dashboard/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "optuna-dashboard" %}
+{% set version = "0.3.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/optuna-dashboard-{{ version }}.tar.gz
+  sha256: 277a81b0ebb33f405dfba2cd15d2ec628abd7b4acacd6364c88cea04ee3fa907
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - optuna-dashboard = optuna_dashboard.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - bottle
+    - optuna >=2.4
+    - python >=3.6
+    - cython
```

Comment:
Does this need cython at run time?

Reply:
Thanks. I have opened a ticket for it https://github.com/conda-forge/optuna-feedstock/issues/23